### PR TITLE
Replace 'counter' function with 'collection.Counter'

### PR DESCRIPTION
**Pull request by edreamleo**
See #813.

### CODE REVIEW

File: pyflakes/checker.py [hunk 1]
```diff
@@ -1773,6 +1761,7 @@ def DICT(self, node):
         # Complain if there are duplicate keys with different values
         # If they have the same value it's not going to cause potentially
         # unexpected behaviour so we'll not complain.
+        counter = collections.Counter
```
Comment (asottile) [inline pyflakes/checker.py:1764]: this should fix the call, this line looks suspicious as written

**Review by asottile — COMMENTED** [review]
please remove "PR:" from your title and commit message.  of course it's a PR because it is

also please remove "all tests pass", we have ci checks in place which will be the judge of that

**Comment by edreamleo** [comment]
@asottile Thanks for your guidance.

File: pyflakes/checker.py (hunk 1 above)
Comment (edreamleo) [reply pyflakes/checker.py]: @asottile rev 2d8f8de does as you requested.

**Comment by edreamleo** [comment]
@asottile I would rather not change a previous commit message, but feel free to change it if think it's best.

**Review by sigmavirus24 — APPROVED** [review]

**Comment by edreamleo** [comment]
@sigmavirus24 Thanks, Ian.

**Comment by sigmavirus24** [comment]
Thanks for helping improve the project, @edreamleo

**Comment by edreamleo** [comment]
@sigmavirus24 You're welcome :-)

**Pull request merged**